content/en/docs/concepts/cluster-administration/networking.md

```diff
@@ -32,6 +32,34 @@ different approach.
 
 To learn about the Kubernetes networking model, see [here](/docs/concepts/services-networking/).
 
+## Kubernetes IP address ranges
+
+Kubernetes clusters require to allocate non-overlapping IP addresses for Pods, Services and Nodes,
+from a range of available addresses configured in the following components:
+
+- The networking addon is configured to assign IP addresses to Pods.
```

Comment:
Hm...
- 74 `network plugin`
- 73 `CNI plugin`
- 16 `CNI provider`
- 12 `network add-on`
- 9 networking plugin
- 5 `Container Network Interface (CNI) plugin`
- 3 `Container Network Interface (CNI) provider`
- 1 network plug-in
- 1 networking addon

(case-insensitive, `content/en/` only, not including this PR)
(It's hard to count instances of using "CNI" as a noun, but those exist too.)

Bleah. Anyway, not "networking addon".

Reply:
I know, I was going to use "network plugin" but then I saw Line73 and preferred to keep it consistent, but I really don't have a strong opinion, if I have to choose I will go with "network plugin"

> for a non-exhaustive list of networking addons supported by Kubernetes.

Reply:
Supporting anecdotal data:

https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/
https://www.google.com/search?q=%22need+a+network+plugin%22+pod

(IMO) some addons provide network plugins; feasibly, one addon could give you a network plugin and something else at the same time (eg: aggregated API server).

Reply:
replaced "networking addon" by "network plugin"